plugins/completion/cmp/default.nix

```
@@ -16,6 +16,45 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   maintainers = [ maintainers.GaetanLepage ];
 
+  description = ''
+    ### Completion Source Installation
+
+    If `plugins.cmp.autoEnableSources` is `true` Nixivm will automatically enable the corresponding source
+    plugins. This is the default behavior, but will work only when this option is set to a list.
+    If you use a raw lua string or set `plugins.cmp.autoEnableSources` to `false`, you will need to explicitly enable the relevant source plugins in
+    your nixvim configuration.
```

I'd be tempted to add a paragraph break here, and maybe use semantic line-wrapping at the comma, but neither are big issues.

```suggestion
    If `plugins.cmp.autoEnableSources` is `true` Nixivm will automatically enable the corresponding source
    plugins. This is the default behavior, but will work only when this option is set to a list.

    If you use a raw lua string or set `plugins.cmp.autoEnableSources` to `false`,
    you will need to explicitly enable the relevant source plugins in your nixvim configuration.
```